### ISSUE TITLE
Wrong note on the *.xyz input format on Wiki

### ISSUE THREAD

**Issue by mondracek** [Probe-Particle/ppafm#218]
Our Wiki says about the *.xyz input: "Don't forget to clear the 2nd line of the xyz format !! " [with the exclamation marks!] but as far as I know, this is no more true after we have improved the functions for reading the geometry. It is even possible to specify lattice vectors in the comment line and in any case, the `loadXYZ` relies on the presence of the comment on the second line.

**Comment by ondrejkrejci** [comment]
Hi @mondracek. I removed that sentence.
Is the _bas_ format still possible? I am speaking mainly about this: https://github.com/Probe-Particle/ppafm/blob/d81e52294b5fdd97942c65884ff7e662146f6fda/ppafm/io.py#L51

I have also added a note about the _POSCAR_ and _CONTCAR_ option.

**Comment by ondrejkrejci** [comment]
@mondracek - I think that this could be closed, but please look at the question above.

**Comment by mondracek** [comment]
Sorry, @ondrejkrejci, I forgot about this issue. Is the _bas_ format the format for `Fireball`? I have not used this for ages, but if anyone is interested, I can ask Pingo if he can give me some `Fireball` input to test.

**Comment by ondrejkrejci** [comment]
Yes please ask Pingo. The usage of xyz makes more sense as it is ase standartized format.

**Comment by ondrejkrejci** [comment]
@mondracek - is there any progress on this? Thank you!
